src/Cake.Issues.Tests/IssueBuilderTests.cs

```diff
@@ -2016,5 +2017,80 @@ public void Should_Set_Run(string run)
                 issue.Run.ToString().ShouldBe(run);
             }
         }
+
+        public sealed class TheWithAdditionalInformationMethod
+        {
+            [Fact]
+            public void Should_Set_Additional_Information()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+
+                // When
+                var issue = fixture.IssueBuilder.WithAdditionalInformation("Foo", "FooValue").Create();
+
+                // Then
+                issue.AdditionalInformation.ShouldContain(new KeyValuePair<string, string>("Foo", "FooValue"));
+            }
+
+            [Fact]
+            public void Should_Throw_When_Key_Is_Used_Multiple_Times()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+
+                // When
+                var issueBuilder = fixture.IssueBuilder.WithAdditionalInformation("Foo", "FooValue");
+                var result = Record.Exception(() => issueBuilder.WithAdditionalInformation("Foo", "AnotherValue"));
+
+                // Then
+                result.IsArgumentException("key");
+            }
+
+            [Fact]
+            public void Should_Throw_If_Key_Is_Null()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+                string key = null;
+
+                // When
+                var result = Record.Exception(() =>
+                    fixture.IssueBuilder.WithAdditionalInformation(key, "FooValue"));
+
+                // Then
+                result.IsArgumentNullException("key");
+            }
+
+            [Fact]
+            public void Should_Throw_If_Key_Is_Empty()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+                var key = string.Empty;
+
+                // When
+                var result = Record.Exception(() =>
+                    fixture.IssueBuilder.WithAdditionalInformation(key, "FooValue"));
+
+                // Then
+                result.IsArgumentException("key");
+            }
+
+            [Fact]
+            public void Should_Throw_If_Key_Is_WhiteSpace()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+                var key = " ";
```

Comment:
What about multiple spaces? And whitespace is not the same thing as a space. You did this test because of the NullWhiteSpaceCheck. That is good. But the name is sometimes missleading. I have not checked that method, but I guess, that whitespace is more than a space. Just a guess, did not check :-)

Reply:
I have done it like the rest of the tests, e. g.  for `Run`